sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/AddressInformation.java

```diff
@@ -53,6 +57,25 @@ public String getProtocolScheme() {
         return this.protocol.scheme();
     }
 
+    public URI getServerKey() {
```

Comment:
the format of the ServerKey returned from here needs to exactly match with format of ServerKey URI constructed by RNTBD. as discussed offline please consider making a class for this.

Reply:
Thanks Mo~
Created a static method in RntbdUtils :: getServerKey to be used across